backend/Persistence/Repositories/RestaurantEFRepository.cs

```diff
@@ -31,15 +30,14 @@ public void Delete(string id)
             _context.SaveChanges();
         }
 
-        public IEnumerable<Restaurant> GetAll()
+        public IQueryable<Restaurant> GetAll()
         {
             return _context.Restaurants.Select(x => x.ToDomain());
         }
 
-        public IEnumerable<Restaurant> GetAllRestaurantsCloserThan(Coords coords, Distances distance)
+        public IQueryable<Restaurant> GetAllRestaurantsCloserThan(Coords coords, Distances distance)
         {
-            return _context.Restaurants.Where(rest => rest.ToDomain().IsCloser(coords, distance))
-                                       .Select(x => x.ToDomain());
+            return GetAll().Where(rest => rest.IsCloser(coords, distance));
```

Comment:
This does not work. 
![image](https://user-images.githubusercontent.com/90202955/144504508-3a8214ff-1da7-46ac-9b11-cd0684704023.png)

Reply:
Do we even use that function anywhere? I think we don't even have api method for that.

Reply:
We never used it since its implementation. It was meant for sorting by distance but was never implemented fully, but we shouldn't have functions that don't work.

Reply:
Fixed